pages/FAQ.md

```diff
@@ -0,0 +1,38 @@
+# FAQ
+
+## I have an error/bug with Slimefun
+Ok, first things first, join our [Support Server](https://slimefun.dev/discord) if you aren't in it already and go to #bug-discussions. We need to determine if it is a bug, user error or intended behaviour.
```

Comment:
Ugh, please just link to the discord server directly or just say "You can find a link at the bottom since it is in the footer".
I'd rather not have thos relayed through that domain on here.